src/tests/cases/test_utils.rs

```diff
@@ -57,6 +57,13 @@ pub fn sample_frames() -> Vec<Box<dyn DataLinkReceiver>> {
             443,
             b"I am a fake tcp download packet",
         )),
+        Some(build_tcp_packet(
+            "10.0.0.2",
+            "1.1.1.1",
+            54321,
+            53,
+            b"I am a fake DNS query packet",
+        )),
```

Comment:
Nice! This way all the other tests test this feature as well, because they should not show this packet.